web-client/src/views/AppComponent.tsx

```diff
@@ -304,6 +305,7 @@ export const AppComponent = connect(
         {showModal === 'WebSocketErrorModal' && <WebSocketErrorModal />}
         {showModal === 'AppMaintenanceModal' && <AppMaintenanceModal />}
         {showModal === 'GenericErrorModal' && <GenericErrorModal />}
+        {showModal === 'FileUploadErrorModal' && <FileUploadErrorModal />}
```

Comment:
This was being called in many different components, often like `<FileUploadErrorModal confirmSeqence={something} />`. However, 1) now this modal is used more often, so it makes sense to centralize it by placing it in the React hierarchy tree here (in fact, I think most modals are best done that way to avoid multiple components potentially rendering the same modal) and 2) `confirmSeqence={something}` was not actually doing anything, so getting rid of it in the components that used it was perfectly safe.